GOVERNANCE.md

```diff
@@ -65,6 +65,48 @@ meeting. In both cases the decision must be seconded by a majority of the Team
 or else the discussion will continue. Simple majority wins. Only Active
 Members participate in a vote.
 
+
```

Comment:
extra newline

Reply:
Should be fixed now.

Reply:
given the amount of markdown we’re working with, probably a good idea to add some linting/auto-formatting tooling. github action maybe?